src/h5web/dataset-visualizer/DimensionMapper.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import ReactSlider from 'react-slider';
+import { Vis, DimensionMapping } from './models';
+import { useVisProps } from './VisProvider';
+import ButtonGroup from '../visualizations/shared/ButtonGroup';
+import styles from './DimensionMapper.module.css';
+
+interface Props {
+  activeVis: Vis;
+  setMapping: (d: DimensionMapping) => void;
+}
+
+function DimensionMapper(props: Props): JSX.Element {
+  const { activeVis, setMapping } = props;
+  const { rawDims, mapping } = useVisProps();
+
+  if (activeVis === Vis.Line && rawDims.length === 2) {
```

Comment:
For now, the apparition of the DimensionMapper is only triggered for LineVis of 2D datasets.